### ISSUE TITLE
`on_mention()` fails when there is a DM

### ISSUE THREAD

**Issue by fotoente** [yupix/Mi.py#84]
**Describe the bug**
When the `in_mention` event is used, the Framwork fails after the routine is finished

**To Reproduce**
Start script
Send a DM to a bot with the `on_mention()` Event activ
Bot will anwser and then gonna fail

**Failure log**
```
Traceback (most recent call last):
  File "/home/misskey/ceilingfox-bot/ceilingfox-reply.py", line 57, in <module>
    asyncio.run(MyBot(prefix).start(uri, token))
  File "/usr/lib/python3.9/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.9/dist-packages/mi/client.py", line 437, in start
    await self.connect(reconnect=reconnect, timeout=timeout)
  File "/usr/local/lib/python3.9/dist-packages/mi/client.py", line 398, in connect
    await self.ws.poll_event()
  File "/usr/local/lib/python3.9/dist-packages/mi/gateway.py", line 50, in poll_event
    await self.received_message(json.loads(msg.data))
  File "/usr/local/lib/python3.9/dist-packages/mi/gateway.py", line 45, in received_message
    self._misskey_parsers[str_lower(msg['type']).upper()](msg)
  File "/usr/local/lib/python3.9/dist-packages/mi/state.py", line 216, in parse_channel
    getattr(self, f'parse_{channel_type}')(base_msg['body'])
AttributeError: 'ConnectionState' object has no attribute 'parse_unread_specified_note'
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f0b90630e20>
Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7f0b9015a3a0>, 15199.94388424)]']
connector: <aiohttp.connector.TCPConnector object at 0x7f0b901886d0>
```

**Comment by yupix** [comment]
The problem is that the parse_unread_specified_note method does not exist, and we plan to fix this in v2.1.2.

**Comment by yupix** [comment]
this problem fixed in a5e7216fa7f8602ce003997a0fe77002f6e279ce
please wait release v2.1.2

**Comment by yupix** [comment]
Fixed
